phc-rails/config/routes.rb

```diff
@@ -4,11 +4,12 @@
   namespace :api do
     namespace :v1 do
       get "search", to: "accounts#search"
-      post "create", to: "accounts#create"
       get "check", to: "accounts#check"
       get "accounts/:sf_id", to: "accounts#show"
-      post 'event_registrations/create'
+      post "create", to: "accounts#create"
+
       get 'event_registrations/search'
+      post 'event_registrations/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.